ops/ansible/playbooks-ccs/templates/awslogs.conf.data-server.j2

```diff
@@ -137,58 +137,58 @@ log_group_name = /var/log/secure
 
 [Jboss_Access_Log]
 datetime_format = %d/%b/%Y:%H:%M:%S
-file = /u01/jboss/jboss-eap-7.0/standalone/log/access*.log
+file = /jboss/wildfly-8.1.0.Final/standalone/log/access*.log
```

Comment:
Should this just use variables so it doesn't need to be updated in the future? E.g., `{{ data_server_dir }}/{{ data_server_container_name }}/standalone/log/access*.log`. *passim*